Add C++ wrapper for shim from_blob in stable/csrc/ops.h

janeyx99 opened the pull request:
One caveat is that I only test the normal from_blob args (so no Layout and no storage_offset)

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #169563
* #169385
* __->__ #168380

janeyx99 committed:
Add C++ wrapper for shim from_blob in stable/csrc/ops.h

[ghstack-poisoned]

janeyx99 committed:
Update on "Add C++ wrapper for shim from_blob in stable/csrc/ops.h"




[ghstack-poisoned]

albanD reviewed (commented):
@mikaylagawarecki any idea?

albanD reviewed (approved):
Sounds good then!

mikaylagawarecki committed:
Update on "Add C++ wrapper for shim from_blob in stable/csrc/ops.h"

One caveat is that I only test the normal from_blob args (so no Layout and no storage_offset)




[ghstack-poisoned]

mikaylagawarecki committed:
Update on "Add C++ wrapper for shim from_blob in stable/csrc/ops.h"

One caveat is that I only test the normal from_blob args (so no Layout and no storage_offset)




[ghstack-poisoned]
